app/External/GitHub/OrganizationApi.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\External\GitHub;
+
+use App\External\ApiProgress;
+use GuzzleHttp\Client;
+use GuzzleHttp\RequestOptions;
+use Illuminate\Support\Collection;
+
+class OrganizationApi
+{
+    use GitHubApiTrait;
+
+    private string $accessToken;
+
+    private Client $client;
+
+    private string $organizationName;
+    private string $organizationUrl;
+
+    /**
+     * TeamApi constructor.
+     */
+    public function __construct($name, string $accessToken)
+    {
+        $this->organizationName = $name;
+        $this->accessToken = $accessToken;
+
+        $this->organizationUrl = "https://api.github.com/orgs/$name";
+        $this->client = new Client();
+    }
+
+    public function team($name): TeamApi
+    {
+        return new TeamApi($this->organizationName, $name, $this->accessToken);
+    }
+
+    public function list(ApiProgress $progress = null): Collection
+    {
+        return $this->paginate("{$this->organizationUrl}/members", function ($url) {
+            return $this->client->get($url, [
+                RequestOptions::HEADERS => [
+                    'Authorization' => "Bearer {$this->accessToken}",
+                    'per_page' => 100,
+                ],
+            ]);
+        }, $progress);
+    }
+
+    public function pending(ApiProgress $progress = null): Collection
+    {
+        return $this->paginate("{$this->organizationUrl}/invitations", function ($url) {
+            return $this->client->get($url, [
+                RequestOptions::HEADERS => [
+                    'Authorization' => "Bearer {$this->accessToken}",
+                    'per_page' => 100,
+                ],
+            ]);
+        }, $progress);
+    }
+
+    public function failed_invitations(ApiProgress $progress = null): Collection
```

Comment:
Are we doing snake_case or camelCase for method names?

Reply:
Right, we should be consistent about camelCase basically everywhere but unit tests. I've just switched between frameworks that have preferences either way in PHP that I sometimes get it wrong. Good catch.